e2e/__tests__/__snapshots__/requireMissingExt.test.ts.snap

```diff
@@ -0,0 +1,34 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`shows a proper error from deep requires 1`] = `
+FAIL ./test.js
+  ● Test suite failed to run
+
+    Cannot find module '../package' from 'node_modules/discord.js/src/index.js'
+
+    However, Jest was able to find:
+    	'../package.json'
+
+    You might want to include a file extension in your import, or update your 'moduleFileExtensions', which is currently ['js'].
+
+    See https://jestjs.io/docs/en/configuration#modulefileextensions-array-string
+
+      1 | 'use strict';
+      2 | 
+    > 3 | require('discord.js');
+        | ^
+      4 | 
+      5 | test('dummy', () => {
+      6 |   expect(1).toBe(1);
+
+      at Resolver.resolveModule (../../packages/jest-resolve/build/index.js:259:17)
+      at Object.<anonymous> (node_modules/discord.js/src/index.js:21:12)
+      at Object.require (test.js:3:1)
```

Comment:
this line is about 15 lines into the stack, which is why it wasn't included before (and we were missing the code frame)